### ISSUE TITLE
WCAG 2.1 AA Accessibility Compliance Epic

### ISSUE THREAD

**Issue by briand44** [uiowa/uiowa#9295]
# Motivation

We need to be WCAG 2.1 AA compliant by April 24th, 2026 to meet the Department of Justice mandate. In addition to compliance, we also want to make sure we are addressing other potential accessibility issues so that our platform and websites are accessible to all.

# Scope and Priority 

- SiteNow public facing websites
  - service wide compliance issues (A & AA) reported in SI
  - site specific compliance issues (A & AA) related to custom functionality
  - Other compliance issues (not found in SI) related to SiteNow public facing websites
  - SI issues (best practices, ARIA) that SiteNow users reporting as they try to increase SI scores
  - Signage public facing compliance issues - Findings are represented in https://github.com/uiowa/uiowa/issues/9156
- UIDS compliance - https://github.com/uiowa/uids/issues/1001
- Icon Browser compliance issues - https://github.com/uiowa/uiowa/issues/9295#issuecomment-3577642235
- SiteNow admin compliance issues - #9413 
- Signage admin?
- SiteNow behind authentication compliance issues

# Additional Info

A11y Milestone - https://github.com/uiowa/uiowa/milestone/6

SiteImprove Cross Site Reporting dashboard - https://my2.siteimprove.com/Dashboard/32195086775/Custom/Dashboard/Index

**Comment by briand44** [comment]
Capturing some notes from our conversation yesterday.

- Should plan work to create SiteNow tag in Siteimprove and use manifest list to compare and tag sites in Siteimprove
  - New SiteNow sites added to Siteimprove should have this tag applied when added
  - Devs can add existing tags but not create new
- Desire to make clear the accessibility work to stakeholders
  - If we are working on it or if we aren't, workaround or they why (e.g. we aren't prioritizing non compliance issues right now)
    - concerns about curating a separate list outside of GH (keeping it up to date)
- Desire to switch focus to admin side of SiteNow while we make public facing compliance issues easier to identify in Siteimprove
  - `node/add`, `node/edit` for content types, content overview, add/edit media, add/edit taxonomy, config pages heavily used
  - Might be able to do a programmatic inventory by scanning module .routing.yml files and matching against  /admin/*
  - identify Drupal core related issues, find any related existing issues, report issues if don't find existing
- Discussed additional training on compliance tool + some clarification of our role vs. site owners role vs. a third party feature’s role vs. the committee’s role, etc. would be helpful as well for campus?
  - Add more documentation and campus related videos on https://itaccessibility.uiowa.edu/siteimprove?
  - https://its.uiowa.edu/services/siteimprove
- Automated testing was mentioned as a possible way to prevent introducing new accessibility issues
- May need to audit outstanding accessibility related issues to understand if they impact compliance - https://github.com/uiowa/uiowa/issues?q=state%3Aopen%20label%3Aa11y
